### ISSUE TITLE
Will this program be updated to the latest patch (1.106.148.1030)?

### ISSUE THREAD

**Issue by Burinis** [Zerbu/Mod-Constructor-5#189]
Will this program be updated to the latest patch (1.106.148.1030)? It's just that there haven't been any updates lately.
If not, are there any forks of Mod Constructor?

**Comment by scipiogarling** [comment]
Indeed: after a year, I'm assuming Zerbu has abandoned this program.

**Comment by EunolaSims4** [comment]
From what he said on Twitter it'll no longer be getting new features just emergency patches as needed
